src/XMLimport.cpp

```diff
@@ -242,6 +242,8 @@ void XMLimport::readVariableGroup(TVar* pParent)
             }
         }
     }
+
+    delete var;
```

Comment:
Had to think about this for a bit, but it does add up - but is this also applicable to any other places where `new TVar()` is used (and not cleaned up afterwards?)